announcements/2020-Week-32/aws.md

```diff
@@ -0,0 +1,16 @@
+# :zap: This Week in AWS Releases (August 7, 2020):zap:
+
+[AWS v11.5.1](https://github.com/giantswarm/releases/tree/master/aws/v11.5.1) fixes a problem with the calculation of the maximum number of Pods per node and improves resource cleanup after cluster deletion.
+
+[AWS v12.1.0](https://github.com/giantswarm/releases/tree/master/aws/v12.1.0) upgrades Cert Manager. (Discussion with SE prior to ugprade is recommended.)
+
+**Optional Apps Releases**  
+(Can be installed from App Catalogs)
+
+1. 
+
+**Web Interface Releases**
+1. 
```

Comment:
See https://gigantic.slack.com/archives/C90BZMUJ1/p1596796237489800 regarding happa v0.10.36

Reply:
@ubergesundheit Can you add the announcement here please?

Reply:
Thanks for the ping!

```suggestion
1. [Happa v0.10.36](https://github.com/giantswarm/happa/releases/tag/v0.10.35) makes cluster upgrades from within happa safer. It makes it no longer possible to trigger an upgrade for non-ready or already upgrading clusters. Additionally, the cluster release selection will no longer contain any empty rows.
```

Reply:
I'll also add the files for azure and kvm

Reply:
@ubergesundheit added happa announcement to all 3 files. I'll do the same for optional apps.